WeScan/Common/RectangleDetector.swift

```diff
@@ -17,18 +18,62 @@ struct RectangleDetector {
     /// - Parameters:
     ///   - image: The image to detect rectangles on.
     /// - Returns: The biggest detected rectangle on the image.
-    static func rectangle(forImage image: CIImage) -> CIRectangleFeature? {
-        let rectangleDetector = CIDetector(ofType: CIDetectorTypeRectangle, context: CIContext(options: nil), options: [CIDetectorAccuracy: CIDetectorAccuracyHigh])
-        
-        guard let rectangleFeatures = rectangleDetector?.features(in: image) as? [CIRectangleFeature] else {
-            return nil
+    static func rectangle(forImage image: CIImage, completion: @escaping ((Quadrilateral?) -> ())) {
+        if #available(iOS 11.0, *) {
+            let imageRequestHandler = VNImageRequestHandler(ciImage: image, options: [:])
+            var biggestRectangle: Quadrilateral? = nil
+            
+            // Create the rectangle request, and, if found, return the biggest rectangle (else return nothing).
+            let rectangleDetectionRequest: VNDetectRectanglesRequest = {
+                let rectDetectRequest = VNDetectRectanglesRequest(completionHandler: { (request, error) in
+                    if error == nil {
+                        guard let results = request.results as? [VNRectangleObservation] else { return }
+                        guard let biggest = results.count > 1 ? results.biggest() : results.first else { return }
+                        
+                        let transform = CGAffineTransform.identity
+                            .scaledBy(x: 1, y: -1)
+                            .translatedBy(x: 0, y: -image.extent.size.height)
+                            .scaledBy(x: image.extent.size.width, y: image.extent.size.height)
+                        
+                        biggestRectangle = Quadrilateral(topLeft: biggest.topLeft.applying(transform), topRight: biggest.topRight.applying(transform), bottomRight: biggest.bottomRight.applying(transform), bottomLeft: biggest.bottomLeft.applying(transform))
+                        completion(biggestRectangle)
+                        
+                    } else { completion(nil) }
+                })
+                rectDetectRequest.minimumConfidence = 0.7 // Be confident.
```

Comment:
The `Be confident.` doesn't really add any value and could be removed

Reply:
This wasn't my contribution.

Reply:
No worries, let's not blame anyone. Either way, would you mind removing it within these code changes? @Boris-Em what are your thoughts here?